CHANGELOG.md

```diff
@@ -1,3 +1,90 @@
+### citus v10.1.0 (June 15, 2021) ###
+
+* Drops support for PostgreSQL 11
+
+* Adds `shard_count` parameter to `create_distributed_table` function
+
+* Adds support for `ALTER DATABASE OWNER`
+
+* Adds support for temporary columnar tables
+
+* Adds `worker_partitioned_table_size`, `worker_partitioned_relation_size`
```

Comment:
Do we really need this item? I think these are not supposed to be used by our users. I'd omit this line as we already have `Optimizes partitioned disk size calculation for shard rebalancer` which is based on these UDFs anyway